src/pms-spec.yaml

```diff
@@ -40,7 +40,7 @@ components:
     accessToken:
       description: Plex Authentication Token
       type: apiKey
-      in: query
+      in: header
```

Comment:
@LukeHagar As discussed, I reverted the earlier change of having the apiKey as a query instead of header